config/projects.yml

```diff
@@ -80,6 +80,13 @@ classification-bundle:
         symfony: ['2.3', '2.7', '2.8', '3.0', '3.1']
         sonata_admin: ['3']
 
+classification-media-bundle:
+  branches:
+    master:
+      php: ['5.3', '5.4', '5.5', '5.6', '7.0']
```

Comment:
Oh you mean code here will be coming from media / classification, and a BC solution should be provided? But this is `master`, right? So you can drop them anyway, at any time, so why not now? Also, why is there only a definition for the master branch?